Assets/MixedRealityToolkit/_Core/EventDatum/Input/SourcePoseEventData.cs

```diff
@@ -91,11 +91,13 @@ public void Initialize(IMixedRealityInputSource inputSource, IMixedRealityContro
         /// </summary>
         /// <param name="inputSource"></param>
         /// <param name="controller"></param>
-        /// <param name="position"></param>
-        public void Initialize(IMixedRealityInputSource inputSource, IMixedRealityController controller, MixedRealityPose position)
+        /// <param name="pose"></param>
+        public void Initialize(IMixedRealityInputSource inputSource, IMixedRealityController controller, MixedRealityPose pose)
         {
             Initialize(inputSource, controller);
-            MixedRealityPose = position;
+            ThreeDofPosition = pose.Position;
```

Comment:
General, non-PR question: is the source pose grip pose, position pose, or both? Do we expose them both somewhere? I couldn't find it yesterday.

Reply:
This is only for the source position. Does not give grip or pointer poses.

You have to implement the IMixedRealityInputPoseHandler to get those.

Reply:
But...where is the "source position" centered? It has to be one of them.

MixedRealityInputPoseHandler didn't seem to distinguish between grip and pointer.

Reply:
> MixedRealityInputPoseHandler didn't seem to distinguish between grip and pointer.

Sure it does, that's what the InputActions are for. You compare the action id to the one you setup via the  event data.

> But...where is the "source position" centered?

That's up to the controller implementation.

Reply:
What if I don't want to use InputActions?

Reply:
Then get the data directly from the controller.